docs/source/tutorials/chat.rst

```diff
@@ -0,0 +1,164 @@
+=================================
+Fine-tuning LLaMA3 with chat data
+=================================
+
+In this tutorial, we'll demystify what prompt templates are and when you'll need them
+and talk about the differences between prompt templating for LLaMA2 and LLaMA3. Then,
+we'll wrap up with a LLaMA3 finetuning example on a custom chat dataset.
```

Comment:
To me this doesn't actually fully line up with the title of the tutorial. Also kind of a nit but I wouldn't use "demystify" in the opening sentence, I think it can potentially turn people off by making a topic seem too complex.

Reply:
Maybe frame it more as, "llama3 handles chat formats differently, here's what you need to know"? Also what would you suggest for the title?